Fix for issue #25 - corrupted display in a few areas when showing/clearing the "long" filename.

frachel opened the pull request.

billkendrick commented:
Making me reconsider my recent _removal_ of POKE()s in my game Gem Drop. :laughing:

tschak909 commented:
Basically, assignments in CC65 are _REALLY_ overdone.

Assignments to arrays are infinitely worse. ;)

jhusak commented:
Well, sometimes they are working well, POKE is the shortcut for smart assignment, everybody knows what it does. So why not use it :)

frachel committed:
Fix 2 issues with long filenames:

1 - clear the 2 bottom lines of the screen where the long filename is displayed when the user selects a file and they are brought to the mount screen. Remnants of the filename were left down there.

2 - Move the MTIME (modified time/date) info down 1 line since there are some filenames that are just really long and wrap 3 lines.

frachel committed:
Merge branch 'FujiNetWIFI:main' into main
